### PR TITLE
Add Electron documentation:

### PR DESCRIPTION
**Pull request by GedasGa**
### Platforms affected

electron

### Motivation and Context

Add Electron documentation to dev section.

### Description

Created an Electron documentation section and added to the sidebar navigation. Also, added Electron to the platform support table, updated custom images documentation and created a link to electron repo.

### Testing

npm t

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [x] I've updated the documentation if necessary

### CODE REVIEW

File: www/docs/en/dev/guide/support/index.md [hunk 1]
```diff
@@ -89,6 +94,7 @@ the core plugins, additional APIs are available via
         <td data-col="ios"     class="y"></td>
         <td data-col="osx"     class="n"></td>
         <td data-col="windows" class="y">Windows Phone 8.1 only</td>
+        <td data-col="electron" class="p">Tests Pending</td>
```
Comment (janpio) [inline www/docs/en/dev/guide/support/index.md]: What does this mean? Either it is known working, or not.

**Comment by janpio** [comment]
All works in local build as expected?

File: www/docs/en/dev/guide/support/index.md (hunk 1 above)
Comment (GedasGa) [reply www/docs/en/dev/guide/support/index.md]: These plugins were not tested. I was wondering maybe the user's should know the actual situation, so I just put "Tests Pending". Well in the case, where is either working or not, I can just say it's not working for now.

**Comment by GedasGa** [comment]
> All works in local build as expected?

The site builds locally fine and works correctly. All tests pass too.

**Pull request merged**